README.md

```diff
@@ -26,8 +26,13 @@ users can know what their official response was.
 Send GitHub the issue URL at the end of the message so that they can
 more easily find updates and further comments here.
 
-Issues will only be closed once GitHub implements / fixes them,
-or explicitly says WONTFIX, which almost never happens nowadays.
+Issues will only be closed in one of the following situations:
+- once GitHub implements / fixes it
+- GitHub explicitly says it's a WONTFIX, which almost never happens nowadays
```

Comment:
This certainly almost never happens. However it's also an extremely difficult task.  I can't say for certainty that GitHub will never implement a given request.  A system that was more flexible would be more possible to work within.  For example I could much more easily mark an issue  <kbd>unlikely</kbd> which may translate to <kbd>wontfix</kbd> for some but for myself more accurately represents the situation.  At this point in time I can make a call about many features but that won't hold true forever.

I'm not trying to nitpick with semantics here but allow a way for me to represent what I know accurately.

Reply:
@clarkbw I think you're saying that very few issues will be closed if not implemented, which actually, basically is a rule of thumb here currently (IMHO).  I also think I understand that you'd like to implement #472, @ least for this repo & those like it, for such otherwise open issues. 🙇

Reply:
>  I also think I understand that you'd like to implement #472, @ least for this repo & those like it, for such otherwise open issues. 🙇

Wait, what? Are you reading my GitHub emails? 😄 

That's likely an ideal solution to this problem.  Might be nice if I could set a "GitHub priority" and there could also be a "Community Priority" that wasn't set by me.

Reply:
Until y'all _do_ have #472 officially accomplished, such could be kludged in via abusing Milestones or even more Labels, I suppose? 🤷🏾‍♂️